en/docs/reference/connectors/pulsar-connector/pulsar-inbound-endpoint-example.md

```diff
@@ -0,0 +1,155 @@
+
+      * **Inbound Functional:**
+        * **Event Integration Name**: `ApachePulsarInboundEP`
+        * **Injecting Sequence Name**: `successSeq`
+        * **Error Sequence Name**: `errorSeq`
+        * **Polling Interval**: `1000`
+        * **Coordination**: `true`
+      * **Pulsar Connection:**
+        * **Enable TLS Encryption**: `true`
+        * **Broker URL**: `pulsar+ssl://localhost:6651/`
+        * **Allow Insecure TLS Connection**: `false`
+        * **Use KeyStore TLS**: `false`
+        * **Broker CA Certificate Path**: absolute path to the CA certificate file (e.g., `/path/to/ca.cert.pem`)
+        * **Authentication Type**: `JWT`
+        * **JWT Token**: the JWT token generated from the Pulsar server (e.g., `eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY`)
+      * **Consumer:**
+         * **Topic(s)**: `persistent://public/default/baa`
+         * **Subscription Name**: `mySubscription`
+         * **Subscription Type**: `Exclusive`
+         * **Subscription Initial Position**: `Latest`
+         * **Message Content Type**: `application/json`
+         * **Message Wait Timeout (in milliseconds)**: `3000`
+         * **Processing Mode**: `sync`
+
+    The source view of the created inbound endpoint is shown below.
+
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <inboundEndpoint name="ApachePulsarInboundEP" class="org.wso2.integration.inbound.PulsarMessageConsumer" 
+    sequence="successSeq" onError="errorSeq" suspend="false">
+        <parameters xmlns="http://ws.apache.org/ns/synapse">
+            <parameter name="interval">1000</parameter>
+            <parameter name="coordination">true</parameter>
+            <parameter name="useTLS">true</parameter>
+            <parameter name="serviceUrl">pulsar+ssl://localhost:6651/</parameter>
+            <parameter name="useKeyStoreTls">false</parameter>
+            <parameter name="tlsTrustCertsFilePath">/path/to/ca.cert.pem</parameter>
+            <parameter name="authenticationType">JWT</parameter>
+            <parameter name="jwtToken">eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY</parameter>
+            <parameter name="topicNames">persistent://public/default/baa</parameter>
+            <parameter name="subscriptionName">mySubscription</parameter>
+            <parameter name="subscriptionType">Exclusive</parameter>
+            <parameter name="subscriptionTopicsMode">PersistentOnly</parameter>
+            <parameter name="subscriptionInitialPosition">Latest</parameter>
+            <parameter name="contentType">application/json</parameter>
+            <parameter name="messageWaitTimeout">3000</parameter>
+            <parameter name="processingMode">Sync</parameter>
+            <parameter name="dlqMaxRedeliverCount">5</parameter>
+            <parameter name="batchReceiveEnabled">false</parameter>
+        </parameters>
+    </inboundEndpoint>
+    ```      
```

Comment:
Let's add a reference to the [Apache Pulsar Inbound Endpoint Reference] page here as well.